BOILERPLATES.md

```diff
@@ -18,3 +18,4 @@
 | [ignite-typescript-boilerplate](https://github.com/aerian-studios/ignite-typescript-boilerplate/) | A boilerplate for TypeScript apps, based on ignite-ir-boilerplate  | [Matt Kane](https://github.com/ascorbic) |
 | [ignite-expo-boilerplate](https://github.com/jbosse/ignite-expo-boilerplate) | Fork of ignite-ir-boilerplate modified to work with Expo.io  | [Jimmy Bosse](https://github.com/jbosse) |
 | [ignite-kryptonite](https://github.com/juddey/ignite-kryptonite) | Your react web thing, ignitified.  | [Justin Lane](https://github.com/juddey) |
+| [ignite-kos-boilerplate](https://github.com/ArtemKosiakevych/ignite-boilerplate-kos) | Fork of ignite-ir-boilerplate-andross  | [Artem Kosiakevych](https://github.com/ArtemKosiakevych) |
```

Comment:
Can you add a summary of what changes you've made?

Reply:
@ArtemKosiakevych Would you want to add the summary here?